src/main/java/com/linkedin/replica/chat/commands/Command.java

```diff
@@ -0,0 +1,29 @@
+package com.linkedin.replica.chat.commands;
+
+import java.util.HashMap;
+
+import com.linkedin.replica.chat.database.handlers.DatabaseHandler;
+
+public abstract class Command {
+
+	public HashMap<String, Object> args;
+	public DatabaseHandler dbHandler;
+
+	public Command(HashMap<String, Object> args) {
+		this.args = args;
+	}
+
+	public abstract String execute();
+	
+	public void setDbHandler(DatabaseHandler dbHandler) {
+        this.dbHandler = dbHandler;
+    }
+
+    protected void validateArgs(String[] requiredArgs) {
+        for(String arg: requiredArgs)
+            if(!args.containsKey(arg)) {
+                String exceptionMsg = String.format("Cannot execute command. %s argument is missing", arg);
+                throw new IllegalArgumentException(exceptionMsg);
```

Comment:
The thrown exception should be a custom exception so that we can identify the difference between a bad request due user's input error and an internal server error. Please add a custom exception class that extends Exception and throw it instead of IllegalArgumentException.